yarn-project/aztec.js/src/abis/ecdsa_account_contract.json

```diff
@@ -48,7 +48,7 @@
MNWXxCO8RhVGiHZ/XXjk5y6D5Hdb9/Ub4fHpP39gvIL773tdC9WuheK3TDMfxcuqCz473xMB2znL5/mWOWh+73LxwzfB5DZ7Sz47OaI5ufhZeQfVGin6vFcY7sq+k4PDucR9twH8LXY+V7uvb7F9zfo+SuTYuIRfm+tn0WuIy0wAUdT/uvIg7le9L22nW58B915psq6zXTiPXKacS6ZBqxXthnVovz1eIZzHW2j2kouevRpeSf8vPNtu9fkFf0L2PL84xn6zK018SzJt3dpgmd8DmBe3jD+8Y2L8wRSi4vfD/3C12WNiZnkBa4oONp/6+Iox/3uL2wntNnVgPdxgndUHLjhll+rsyC8xk5QP84n8HJ8zs/I1s+N4Vz73RxXOA+TZnbCY5dxfA0YvmpLkt77p3a2TE+0PG0/wLiOFWXox3PPxX+o858U2U9vc+sBrlqv3/5iW6f7VjkmIaSG4s/If9+rMyC8xl5Rf8ytnw+/5Bs7Ofz+UfiuMD9A2VuJzh2FcMfEMspuizt+XxyZ8f4QMfT/vOJ42RdjvYcOUX4jzrzTZX1R31mNchV+/3LSbp9tmORYxpKbiyeRP59X5kF5zPyiv5lbPl8/h7Z2M/n84niOE/H833793R9Gc/FMMVy/QCxjAwQy8wBYjlxgFhGB4hl8QCxzBgglgsHiOWSAWJxA8QyPEAsswaIZfYAsVR9ZnGdHe+JHO2/kLYNibbh8+u4PSb3nxC3DyX6OaGz43Hs+/EGvrPOBNWhNUoMJwwAy+wBYpk1QCzDA8TiBojlkgFiuXCAWGYMEMviAWIZHSCWEweIZeYAsYwMEMv1A8QylGA5TpelnS9c2JksuHc5jjjAtJA4FijHJPRxbIJjAXFA/1jiOEaXo30PzdEJjmOIA/pHE8dRuhxznOAIJXdvexSxfFeXpR0jR5IWuKDjaf/xxHGkLkd73nxX+I86802VdeE0Yj22z6wG46p9b8V3dPts5785pqHkzpvvkH/f1mWZg/lv5BX9y9gOEc98srGfr8XfShwHewbtx73pMO1/cXwID2s/72zefL5uDLLP49Da2bx5v1lGBohl5gCxnDhALKMDxLJwgFgWDxDLjAFiOX6AWC4ZIBY3QCzDA8Qya4BYZg8QS9Vnlp3N4WP/8bTtiGgvpG1Dif4w7nB8+Mw9meb6D4/bea7/sGgPJ/QOT3AdlmjLsUSbifh37JmVNpasM0F1aPF3AocNAMvsAWKZNUAswwPE4gaI5ZIBYjl+gFhmDBDL4gFiWThALKMDxHLiALHgeXcQWEYGiOX6AWIZSrB8U5el4fuuDjFxmSAb+uG2LdxH4F5svuDnOa9DdZnb+fKvExM0D6XYfSPafBzscN3G5wiOG6b9P6N5vMB/iC5/+13MwQn+Q4gfLHzcwcS/UPg0TPvPEvxf0+VvvzthrlByYwb6geUgXZZ2LBzY2TFG0PG0n+fVDtTlaM/pg4T/qDPfVFmPmEas8/vMajGuQp9f1e2zvdZyTEPJnTdfJf8O0GXZ9t0J8or+ZWyHiGce2djP19z9xXGB+yu63Hs5wbGrGH6FWObqsrTXnv06O8YHOp728/cu++lytOfIXOE/6sw3VdbF04j1kmnEun+fWR1tm0fbhgTzrI7dOjJYC6bqbL/uCwqvkXNx/HsdHSfXw+H1La+h47DWBa8PgzVQeG2pK+PfIernKtp/BdlLRJugvUjohLj1Y/1CXpftCkMW39n1Oo8Xxb+cN9mG1zFcRMddFu1zaRvWrOC8YW2JIernUtp/DtmXiTa8NgOvFaS9dljV2X4NiAmqQ2tn69xps/iE9iht47XNVNefGO9+TkJzqLPjmg4hH2ckOJTXM2jzcYbIB+o/p3yk1ljTZvEJbQudjvB5VGjyum44Tq7rZvE+F7yTpaL+HXGuIBvve3mYfFgpjnNkr6Tjlkeb39OCd5HwO57ujX+HqJ/7af89ZC8XbYL2UqET4qb8/oZ2/C4ljgmq8zss7hTMFiw+oS1tvF+H8ybbOLKX0nF4b8JttA3vjuC84R0PQ9TPHbT/VrLvEm34HQn8zh7td3hVne3fxTBBdWjt7H1z2iw+oT1K23j90Js0tePnADSHOju+WyHk48YEh/J7Bdp83CjygfoNlI/Uu860WXxC20KnI3x2QpPfr8bv4sD71cIY+W3cvok4nzTg3Eyc+K76SdJ8XFezXZvzCdIM/eIzCVoV2SPxGHA+QdsfVY5H0H0s9jVTcEBrmI55Rfw9RR3rGzuTBetng3uU2m0kHzao+tC07w9lDsSTywTZG4hlnSpLN9ePUP8TpMG6a3V1x1m3iv+gge3DZO+1x2Q81k6a285BMIccrk8cx/ZG0cbT/vXGPq8jjgmqQytcY15Dvq5PcM8mbuzHuA1jC3nj69J6A182dLb3ZYNg9sSw0Yyl+/53qe0oTpsptlviRr5+byE26+s3ft/zbF+/cR5Nt+v3+zLXb5wzu/v1ew31P9FJX79X6+qOsy6u39DA9mGyP0PXtNWTZvb6zcexnbp+Y/96Y593dv2GVrjGfGgX1+9ZxJ27fvN1qV/X7y2C/dm8fiNOfP3+NF2/t8btT3Umy2/J5v/DgjZbxHEj1P+Eij/dNdGZI5Tc9WEzsWzSZdnuswDXh00JXeXPoHHWxfUBGtg+TPZ8OmcenzS35RjMIYdPJo5jWz6Xedpv+Lnb+ryJOCaoDq0whg8iX59McI8QN/Y/m/coI4LNkR9P0TFPPIvxe4Ji8lQiXsose/uEtutMxoavSTNmTXLM0OWYw/9PACV3LYF+uEximB0w74iPzN/v4APn7jvvyIr6QL/DiX6HyOeZ4jj+jhT78B3pTF3/29+IzOhM3f+Zaf8/Nm/uYfOOEP7PEL4OdXb0n/0e6Ux+Jk3o+DcGVpQZFM9Q+Peg/P9xnC7HOLQkhyOO2WRjH/9/SGzzgpfHC7dFXD35LHOr5iBgqggzHJ2c2ZlM7OydBGA0QtJ1u/N/ZckcJ7ksAgA=",
```

Comment:
not sure why this bytecode changed when i rebuilt noir-contracts.  m1 difference?  will retry on mainframe